nucypher/cli/characters/stake.py

```diff
@@ -63,6 +63,7 @@
 @click.option('--lock-until', help="Period to release re-staking lock", type=click.IntRange(min=0))
 @click.option('--beneficiary-address', help="Address of a pre-allocation beneficiary", type=EIP55_CHECKSUM_ADDRESS)
 @click.option('--allocation-filepath', help="Path to individual allocation file", type=EXISTING_READABLE_FILE)
+@click.option('--all', help="List all stakes, including inactive", is_flag=True)
```

Comment:
`--all` sounds like a too general word for this option. Maybe `--show-inactive`, or `--with-inactive`?

Reply:
I agree, although @szotov told me something today about how this are not really "inactive", so perhaps we have to rethink this option altogether.

Reply:
Yeah, this needs a rename. Does this have an issue open?